modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} update process.
+
+. The intent to update the cluster is expressed through the web console or the CLI.
+The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource.
+
+. The Cluster Version Operator (CVO) discovers that the `desiredUpdate` in the ClusterVersion resource differs from the current cluster version.
+Using the graph data from the OpenShift Update Service, the CVO resolves the desired cluster version to a pullspec to the release image.
```

Comment:
only true for `--to` not for `--to-image` which sets .desiredUpdate.image directly? however may be something to omit as well.. @wking wdyt?

Reply:
I'm interested in what others have to say, but personally: I think there are so many alternative options/configurations/mechanisms for updates that this page might be best if it only described how a generic, typical update works, and have pages that deal with exceptions (EUS, restricted network, etc.) elaborate the differences there. Trying to describe the possible exceptions on this page might be confusing and overwhelming for users trying to understand updates, even if this is just one exception. 

That's just my opinion though, and I'm happy to hear out counterpoints

Reply:
`--to-image` is a poweruser thing that we do not want to stress out, IMO

Reply:
> this page might be best if it only described how a generic, typical update works

makes sense

Reply:
`modules/update-evaluate-availability.adoc` is already carrying the data from the update service to `availableUpdates` and `conditionalUpdates`.  Once we get to the point of processing a `desiredUpdate` request, [the CVO only looks at `availableUpdates`][1], and folks requesting a supported-but-not-recommended target will [set `image` (and optionally `version`)][2], even if they are using the `--allow-not-recommended --to "${VERSION}"` phrasing.

But one way to keep all of this up at the high-concept level is to keep out of these sub-properties, and say something like:

```asciidoc
. The target release is indicated via the `spec.desiredUpdate` field of the ClusterVersion resource, which may be managed through web console or the CLI.

. The Cluster Version Operator (CVO) discovers that the `desiredUpdate` in the ClusterVersion resource differs from the current target release, and it resolves `desiredUpdate` into a release image pullspec.
```

[1]: https://github.com/openshift/cluster-version-operator/blob/f74d114b1b2843f045fdcdc63ca63fa4cf07da75/pkg/cvo/updatepayload.go#L364-L375
[2]: https://github.com/openshift/oc/blob/603c3afe81f7e705000d1505610314a819ddfc0d/pkg/cli/admin/upgrade/upgrade.go#L282-L283